prov/efa/src/efa_prov_info.c

```diff
@@ -40,30 +37,6 @@ const struct fi_fabric_attr efa_fabric_attr = {
 	.prov_version	= OFI_VERSION_DEF_PROV,
 };
 
-/**
- * @brief set the fabric_attr field of a prov_info
- *
- * @param 	prov_info[out]		prov_info object
- * @param	device[in]		pointer to an efa_device struct, which contains device attributes
- * @return	0 on sucessess
- * 		-FI_ENOMEM if memory allocation failed
- */
-static
-int efa_prov_info_set_fabric_attr(struct fi_info *prov_info, struct efa_device *device)
```

Comment:
This function can be reused by introducing fabric_name as an input? It doesn't seem you need `device` here.